contrib/docker/libvirt/start.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -o errexit
+
+modprobe kvm
```

Comment:
Ok, this also could be used as check, but still intention seems to be not specified - maybe describe this in way similar to:
```
modprobe kvm || echo "Missing kvm module on host" && exit 1
```